locales/zh-Hant/pokemon.json

```diff
@@ -54,32 +54,32 @@
       }
     },
     "karp": {
-      "orange": "",
-      "white": "",
-      "gold": "",
-      "brown": "",
-      "pink": "",
-      "blue": "",
-      "grey": "",
-      "purple": "",
-      "apricot": "",
-      "black": "",
-      "violet": "",
-      "skelly": "",
-      "calico": "",
-      "twoTone": "",
-      "orca": "",
-      "dapples": "",
-      "bubbles": "",
-      "diamonds": "",
-      "patches": "",
-      "tiger": "",
-      "zebra": "",
-      "stripes": "",
-      "forehead": "",
-      "mask": "",
-      "saucy": "",
-      "raindrops": ""
+      "orange": "紅",
+      "white": "白",
+      "gold": "金",
+      "brown": "褐",
+      "pink": "粉紅",
+      "blue": "藍",
+      "grey": "灰",
+      "purple": "紫",
+      "apricot": "橙",
+      "black": "黑",
+      "violet": "淺紫",
+      "skelly": "魚骨",
+      "calico": "錦鯉",
+      "twoTone": "雙色",
+      "orca": "虎鯨",
+      "dapples": "波紋",
+      "bubbles": "水珠",
+      "diamonds": "菱形",
+      "patches": "長頸鹿",
+      "tiger": "老虎",
+      "zebra": "斑馬",
+      "stripes": "豎條",
```

Comment:
The wiki link has this as `竖条` which looks like the simplified version - mistake on that wiki?

Reply:
No it's correct. I was too hasty in my descriptions, the website is actually in simplified and I used external sites to translate to traditional! You can tell 豎條 is correct because it looks much less... simplified!